unicodetools/data/ucd/dev/UnicodeData.txt

```diff
@@ -35658,6 +35658,20 @@ FFFD;REPLACEMENT CHARACTER;So;0;ON;;;;;N;;;;;
 1DF28;LATIN SMALL LETTER R WITH MID-HEIGHT LEFT HOOK;Ll;0;L;;;;;N;;;;;
 1DF29;LATIN SMALL LETTER S WITH MID-HEIGHT LEFT HOOK;Ll;0;L;;;;;N;;;;;
 1DF2A;LATIN SMALL LETTER T WITH MID-HEIGHT LEFT HOOK;Ll;0;L;;;;;N;;;;;
+1DF2D;LATIN SMALL LETTER D WITH HOOK AND PALATAL HOOK;Ll;0;L;;;;;N;;;;;
+1DF2E;LATIN SMALL LETTER DZ DIGRAPH WITH PALATAL HOOK;Ll;0;L;;;;;N;;;;;
+1DF2F;LATIN SMALL LETTER ETH WITH PALATAL HOOK;Ll;0;L;;;;;N;;;;;
+1DF30;LATIN LETTER SMALL CAPITAL G WITH STROKE AND PALATAL HOOK;Ll;0;L;;;;;N;;;;;
```

Comment:
Has this name gained an extra `WITH STROKE`?

Reply:
Hmm, it seems it lost it between the document that was seen in SAH (the one dated February 2 on the SAH issue) and the one posted to the registry.

Great that Kirk corrected his mistake in his UnicodeData lines (the name was correct elsewhere in the proposal, this is really a typo), but it would be nice of him to call that out…